ui/test/nuxeo-collapsible.test.js

```diff
@@ -0,0 +1,132 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { html, fixture, flush, isElementVisible, tap } from '@nuxeo/testing-helpers';
+import '../widgets/nuxeo-collapsible.js';
+import '../nuxeo-icons';
+
+suite('nuxeo-collapsible', () => {
+  suite('Properties', () => {
+    test('Should display nuxeo-collapsible when declaring the element', async () => {
+      const nuxeoCollapsible = await fixture(
+        html`
+          <nuxeo-collapsible></nuxeo-collapsible>
+        `,
+      );
+      await flush();
+
+      expect(isElementVisible(nuxeoCollapsible.shadowRoot.querySelector('button'))).to.be.true;
+    });
+
+    test('Should display heading when it is set', async () => {
+      const nuxeoCollapsible = await fixture(
+        html`
+          <nuxeo-collapsible>
+            <span class="heading" slot="heading">Content</span>
+          </nuxeo-collapsible>
+        `,
+      );
+      await flush();
+
+      const heading = nuxeoCollapsible.shadowRoot.querySelector('slot').assignedElements()[0];
+      expect(isElementVisible(heading)).to.be.true;
+      expect(heading.innerHTML).to.equal('Content');
```

Comment:
Here we check the slot content but on the following tests we don't.
Guess we should try to have consistency on this 😉

Reply:
What is your idea here? To repeat this check in every single test? Wouldn't that be over testing?

Reply:
I was just stating that we've checked `heading` slot content but we never checked the other slot content (just that it is visible). Or am I missing something?

Reply:
I was interpreting your comment differently. I will add that verification on tests in line 60 and 76.